src/main/java/net/consensys/wittgenstein/protocols/ETHPoW.java

```diff
@@ -86,6 +86,7 @@ public void init() {
     final long difficulty;
     final BigInteger totalDifficulty;
     final List<Transactions> transInBlock;
+    final List<Long> uncles = new ArrayList<>();
```

Comment:
It should be the Block itself rather than its id: it's simpler but as well we need the block to calculate the total difficulty in the next blocks.

Reply:
well I only included de the id because in the real ethereum when you create a block with uncle you only store their headers as a hash.